src/main/msp/msp_serial.h

```diff
@@ -27,7 +27,7 @@
 #include "msp/msp.h"
 
 // Each MSP port requires state and a receive buffer, revisit this default if someone needs more than 3 MSP ports.
-#define MAX_MSP_PORT_COUNT 3
+#define MAX_MSP_PORT_COUNT 4
```

Comment:
Maybe this way:
```suggestion
#ifndef MAX_MSP_PORT_COUNT 
#define MAX_MSP_PORT_COUNT 3
#endif
```

Reply:
yes, such would allow custom define.
@kensherman, please commit this suggestion, then pilots may do this: 
![image](https://github.com/user-attachments/assets/5d53d6d0-b27a-4460-9daa-13e218dfbb24)